meteor-app/server/api/v1/publications/publications.ts

```diff
@@ -12,6 +13,15 @@ Meteor.publish('AccountViewers', function () {
 	}
 })
 
+Meteor.publish('AccountTags', function() {
+	if (this.userId) {
+		let account = getAccount(this.userId)
+		return account?.tags
```

Comment:
So you cannot return a non cursor from a publication. 

So returning anything that isn't a .find() will result in error